blivet/iscsi.py

```diff
@@ -414,6 +417,14 @@ def log_into_node(self, node, username=None, password=None,
                      node.name, node.address, node.port, node.iface)
             if not self._mark_node_active(node):
                 log.error("iSCSI: node not found among discovered")
+            if username:
+                node.username = username
+            if password:
+                node.password = password
+            if r_username:
+                node.r_username = r_username
+            if r_password:
+                node.r_password = r_password
```

Comment:
A don't this can work. A ``NamedTuple`` is immutable, AFAICT. If we need to do this, we need to change the ``NodeInfo`` into a traditional class.